src/main/java/WayofTime/bloodmagic/item/alchemy/ItemLivingArmourPointsUpgrade.java

```diff
@@ -63,6 +63,14 @@ public ItemStack onItemUseFinish(ItemStack stack, World worldIn, EntityLivingBas
                 ItemStack chestStack = Iterables.toArray(player.getArmorInventoryList(), ItemStack.class)[2];
                 LivingArmour armour = ItemLivingArmour.getLivingArmour(chestStack);
                 if (armour != null) {
+                    if (player.capabilities.isCreativeMode) {
```

Comment:
This disallows creative players from creating a normal set of armor (eg: server admin recreating items a player lost).

This block and the one after could also be condensed since all that's changing is the amount.